cmd/clusterawsadm/cmd/root.go

```diff
@@ -92,7 +93,7 @@ func Execute() {
 
 func init() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
-	verbosity := flag.CommandLine.Int("v", 2, "Set the log level verbosity.")
+	verbosity = flag.CommandLine.Int("v", 2, "Set the log level verbosity.")
```

Comment:
I don't think this was ever working before properly. :D 
This was trying to use the package local variable but created a new one instead. So